csrc/pf_guts.h

```diff
@@ -63,10 +80,19 @@
 #define FTRUE (-1)
 #define BLANK (' ')
 
-#define FLAG_PRECEDENCE (0x80)
+/* #define FLAG_PRECEDENCE (0x80) */
```

Comment:
Please add:
/* The IMMEDIATE flag is known as the "precedence bit" on some other Forth systems. */